state/aws/dynamodb/dynamodb.go

```diff
@@ -66,13 +69,16 @@ func (d *StateStore) Init(metadata state.Metadata) error {
 	d.client = client
 	d.table = meta.Table
 	d.ttlAttributeName = meta.TTLAttributeName
+	seed := (uint64(os.Getpid()) << 48)
```

Comment:
Please use `crypto/rand` instead of `math/rand`. It doesn't require initialization AND it's actually a better PRNG.

Reply:
fixed. i considered this initially but i didn't need cryptographic level randomness for correctness and my understanding was that math/rand was more performant. i see though that golangci-lint is also flagging this and i don't have a strong preference.

Reply:
`math/rand` is deterministic and that's not really a good thing in general. Performance of `crypto/rand.Reader` should not be observably worse, especially on modern kernels.

And yes, our linters don't like `math/rand` too :)